README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
```

Comment:
Add an Objective to game. For example, "Main objective is to stack different shaped blocks with out over stacking. Thus there is no end to the game and you want to achieve the highest score possible"

Reply:
great Idea